src/transforms/push.ts

```diff
@@ -154,7 +162,7 @@ export const processPush = async (github: GitHubInstallationClient, payload: Pus
 						authorTimestamp: githubCommitAuthor.date,
 						displayId: commitSha.substring(0, 6),
 						fileCount: files.length, // Send the total count for all files
-						files: filesToSend.map((file) => mapFile(file, repo, owner.name, sha.id)),
+						files: compact(filesToSend.map((file) => mapFile(file, repo, sha.id, owner.name))),
```

Comment:
i'm not familiar with `compact` i assume it gets rid of the undefined?

Reply:
correct!